### PR TITLE
migrate `local-environment` page to Chakra

### PR DESCRIPTION
**Pull request by soheil555**
## Description

Migrate `local-environment` page to Chakra

## Related Issue
#9353

### CODE REVIEW

**Comment by soheil555** [comment]
@pettinarip What should be done with the commented codes?
Should they be migrated to Chakra?

**Comment by pettinarip** [comment]
> @pettinarip What should be done with the commented codes? Should they be migrated to Chakra?

Yes please, lets remove those commented components, imports, and jsx 🙏🏼 they were commented 2 years ago xD

**Comment by soheil555** [comment]
@pettinarip Sorry for my confusion but It's not clear to me what to do with them. Should they be removed or migrated?

File: src/pages/developers/local-environment.tsx [hunk 1]
```diff
@@ -1,279 +1,48 @@
...
 import PageMetadata from "../../components/PageMetadata"
 import ProductCard from "../../components/ProductCard"
 import {
   Content,
   CardGrid,
-  // Divider,
-  // Intro,
   Page,
-  // InfoBanner,
 } from "../../components/SharedStyledComponents"
```
Comment (pettinarip) [inline src/pages/developers/local-environment.tsx:24]: As a quick review, one thing we want to deprecate is the usage of `SharedStyledComponents`. We need to replace those components with Chakra components.

**Review by pettinarip — CHANGES_REQUESTED** [review]
Thanks @soheil555 for cleaning this up.

File: src/pages/developers/local-environment.tsx (hunk 1 above)
Comment (soheil555) [reply src/pages/developers/local-environment.tsx]: I'm sorry, it's been resolved.

Comment (pettinarip) [reply src/pages/developers/local-environment.tsx]: Nice! no need to say sorry. Thank you for the great work!

**Review by pettinarip — APPROVED** [review]
lgtm, gj @soheil555 🚀

**Comment by donatocayurin** [comment]
❤️🙏

**Pull request merged**